test/providers/dnsimple_test.rb

```diff
@@ -328,6 +328,21 @@ def test_apply_sshfp_changeset
         provider: RecordStore::Provider::DNSimple,
         zone: @zone_name
       ))
+
+      records = @dnsimple.retrieve_current_records(zone: @zone_name)
+
+      matching_records = records.select do |record|
+        record.is_a?(Record::SSHFP) &&
+          record.fqdn == sshfp_record.fqdn
+      end
+
+      assert_equal(1, matching_records.size, 'could not find the SSHFP record that was just created')
+      assert_equal(Record::SSHFP::Algorithms::ED25519, matching_records.first.algorithm)
+      assert_equal(Record::SSHFP::FingerprintTypes::SHA_256, matching_records.first.fptype)
+      assert_equal(
+        '4e0ebbeac8d2e4e73af888b20e2243e5a2a08bad6476c832c985e54b21eff4a3',
+        matching_records.first.fingerprint
+      )
```

Comment:
Yes, it's unrelated, but deliberately so. I thought I could sneak in some changes here to strengthen tests from the [SSHFP pull request](https://github.com/Shopify/record_store/pull/164) as I noticed they weren't really asserting that any records had been persisted.

I'm willing to just back this change out. The response is stored on `cassette` anyway so it's really only verifying a change took place on the remote server the very first time it's run.

Reply:
I'm fine with the occasional drive-by  :-)

mostly making sure I wasn't missing something, and can see the commit now

for future, maybe the commit comment could be more description for cases like this, or a note added to the PR description to call out the out-of-scope change -- ie. the comment "Assorted test cleanup" didn't clearly tell me what it was doing, and if I were looking at this and doing some archaeology in future it might be confusing